src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
```

Comment:
We use ${({ theme }) => theme.breakpoint.mobileMax a lot. Maybe let's make `const bpMobile = ({ theme }) => theme.breakpoint.mobileMax;`?

Reply:
roger that